src/FSharp.Control.TaskSeq.Test/TaskSeq.Append.Tests.fs

```diff
@@ -95,6 +95,7 @@ module SideEffects =
 
     [<Theory; ClassData(typeof<TestImmTaskSeq>)>]
     let ``TaskSeq-appendSeq consumes whole sequence once incl after-effects`` variant =
+        // TODO use variant
```

Comment:
Yeah, this is just a copy-paste mistake, it should not be a `Theory`, it is just a test that checks if mutable side effects get executed. Same for the other ones.

Reply:
Feel free to take this PR forward, or attack it over time - it's just me addressing my obsessive removal of stuff the Rider linting flags and needing to tell someone or something to park it!